markdown/GameGuides/RoboCopRogueCity.md

```diff
@@ -0,0 +1,58 @@
+![Hogwarts Legacy](Images\robocop_rogue_city_header.png "Shot by Dogmod"){.shadowed .autosize}
+## Summary
+Feature | Supported
+--|--
+Vanilla Photo Mode | No
+Hotsampling | Yes (use windowed fullscreen)
+DSR | No
+Custom Aspect Ratios | Yes  
+Reshade | Yes
+Ansel | No
+Graphics API | DirectX 12
+Game version |  <font face="Stores">S</font> 1.1.1.0
+
+@alert neutral  
+This guide is for an Unreal Engine 5 game that can use the [Universal Unreal Unlocker](https://opm.fransbouma.com/uuuv5.htm).  
```

Comment:
Not sure if linking to Otis's site is the right thing to do in this case, but i couldn't find UUU5 on framed.

Reply:
That's fine